site/en/guides/border-router/build.md

```diff
@@ -63,6 +75,12 @@ To use Wi-Fi:
 $ INFRA_IF_NAME=wlan0 ./script/setup
 ```
 
+To use Network Manager for [BeagleBone Black](beaglebone-black.md) (optional):
+
+```
+$ NETWORK_MANAGER=1 NETWORK_MANAGER_WIFI=1 INFRA_IF_NAME=wlan0 ./script/setup
```

Comment:
I think the `INFRA_IF_NAME` option is meant as an input to the border routing option, which won't work with network manager. And the wlan0 interface in that case is offering a downstream connection to the stations connected to the AP.

Reply:
Ah - OK thanks, I took it out. Thanks for taking a look....